recipes/gcc/all/conanfile.py

```diff
@@ -1,126 +1,112 @@
-            return self._autotools
-        self._autotools = AutoToolsBuildEnvironment(self)
-        pkgversion = 'conan GCC %s' % self.version
-        bugurl = self.url + '/issues'
-        libdir = "%s/lib/gcc/%s" % (self.package_folder, self.version)
-        args = [
-            "--enable-languages=c,c++",
-            "--disable-nls",
-            "--disable-multilib",
-            "--disable-bootstrap",
-            "--with-system-zlib",
-            "--with-gmp=%s" % self.deps_cpp_info['gmp'].rootpath,
-            '--with-mpc=%s' % self.deps_cpp_info["mpc"].rootpath,
-            "--with-mpfr=%s" % self.deps_cpp_info["mpfr"].rootpath,
-            "--without-isl",
-            "--libdir=%s" % libdir,
-            '--with-pkgversion=%s' % pkgversion,
-            "--program-suffix=-%s" % self.version,
-            "--with-bugurl=%s" % bugurl
-        ]
-        if self.settings.os == "Macos":
-            xcrun = tools.XCRun(self.settings)
-            args.extend([
-                '--with-native-system-header-dir=/usr/include',
-                "--with-sysroot={}".format(xcrun.sdk_path)
-            ])
-        self._autotools.libs = []  # otherwise causes config.log to fail finding -lmpc
-        if self.settings.compiler in ["clang", "apple-clang"]:
-            # xgcc: error: unrecognized command-line option -stdlib=libc++
-            if self.settings.compiler.libcxx == "libc++":
-                self._autotools.cxx_flags.remove("-stdlib=libc++")
-            elif self.settings.compiler.libcxx in ["libstdc++", "libstdc++11"]:
-                self._autotools.cxx_flags.remove("-stdlib=libstdc++")
-        self._autotools.configure(args=args, configure_dir=self._source_subfolder)
-        return self._autotools
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
 
-    def requirements(self):
-        self.requires("mpc/1.2.0")
-        self.requires("mpfr/4.1.0")
-        self.requires("gmp/6.2.0")
-        self.requires("zlib/1.2.11")
-
-    def configure(self):
-        if self.settings.os == "Windows":
-            raise ConanInvalidConfiguration("Windows builds aren't supported (yet), sorry")
-        if tools.cross_building(self.settings):
-            raise ConanInvalidConfiguration("no cross-building support (yet), sorry")
+class GccConan(ConanFile):
+    name = 'gcc'
+    description = 'The GNU Compiler Collection includes front ends for C, ' \
+                  'C++, Objective-C, Fortran, Ada, Go, and D, as well as ' \
+                  'libraries for these languages (libstdc++,...). '
+    topics = 'gcc', 'gnu', 'compiler', 'c', 'c++'
+    homepage = 'https://gcc.gnu.org'
+    url = 'https://github.com/conan-io/conan-center-index'
+    license = 'GPL-3.0-only'
+    settings = 'os', 'compiler', 'arch', 'build_type'
+    requires = 'mpc/1.2.0', 'mpfr/4.1.0', 'gmp/6.2.1', 'zlib/1.2.12', 'isl/0.24'
+    tool_requires = 'flex/2.6.4'
+    short_paths = True
+
+    _auto_tools = None
```

Comment:
This should no longer be cached with the new ones

Reply:
thanks